packages/graphback-runtime-mongodb/src/utils/createIndexes.ts

```diff
@@ -85,4 +95,20 @@ export function getRelationIndex(field: GraphQLField<any, any>): IndexSpecificat
   } else {
     return undefined;
   }
+}
+
+export function isCustomIdField(field: GraphQLField<any, any>): IndexSpecification {
```

Comment:
This is more like like `getIndexForPrimaryKey` since it is returning the index?

Reply:
Yes, I'll fix that, thanks!